services/domains/domains.go

```diff
@@ -0,0 +1,56 @@
+package domains
+
+import (
+	CoinType "github.com/trustwallet/blockatlas/coin"
```

Comment:
did you consider rename coin to cointype ? to be `CoinType`

Reply:
no, this name comes from the current implementation, but I prefer to call this package `coins`, in the plural